0-quickie-plugin-scraper/src/main/java/org/pinkcrazyunicorn/quickie/plugins/scraper/CachedDownloader.java

```diff
@@ -0,0 +1,57 @@
+package org.pinkcrazyunicorn.quickie.plugins.scraper;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class CachedDownloader {
+    private final String baseUrl;
+
+    public CachedDownloader(String baseUrl) {
+        super();
+        this.baseUrl = baseUrl;
+    }
+
+    public Document getDocument(String url) throws IOException {
+        return this.getDocument(url, true);
+    }
+
+    public Document getDocument(String url, boolean useCache) throws IOException {
```

Comment:
Wird useCache in Zukunft noch benutzt? Im Moment ist es nutzlos.

Reply:
Es wird aktuell bereits verwendet. Die recipes werden mit useCache=True heruntergeladen und die Kategorieseiten mit useCache=False